internal/tf/import.go

```diff
@@ -34,3 +34,13 @@ func ValidateResourceIDPriorToImportThen(idParser ResourceIDValidator, importer
 		},
 	}
 }
+
+func ImportNotSupported() *schema.ResourceImporter {
```

Comment:
(as above) this isn't necessary - Terraform supports this natively if we remove this